xcarchive/ios.go

```diff
@@ -109,7 +110,7 @@ func NewIosWatchApplication(path string) (IosWatchApplication, error) {
 	}
 
 	extensions := []IosExtension{}
-	pattern := filepath.Join(path, "PlugIns/*.appex")
+	pattern := filepath.Join(regexp.QuoteMeta(path), "PlugIns/*.appex")
```

Comment:
applying `regexp.QuoteMeta` on `path` or wiring in a `patternPrefix` var instead of using `regexp.QuoteMeta` everywhere is a workable way?